cypress/support/commands.js

```diff
@@ -0,0 +1,29 @@
+Cypress.Commands.add('login', (sanitySessionToken) => {
+  const token = sanitySessionToken ? sanitySessionToken : Cypress.env('SANITY_SESSION_TOKEN')
+
+  cy.intercept({url: 'v1/users/me', method: 'GET'}).as('getUser')
+
+  return cy.visit('/').then(() => {
+    cy.wait('@getUser').then((interception) => {
+      const domain = new URL(interception.response.url).hostname
+
+      cy.setCookie('sanitySession', token, {
+        secure: true,
+        httpOnly: true,
+        sameSite: 'None',
+        domain: `.${domain}`,
+      })
+    })
+  })
+})
+
+Cypress.Commands.add('getField', (fieldName) => {
+  // TODO(@benedicteb, 2021-01-26) Add <data-qa=..." /> or something to html to make select super robust
```

Comment:
(Side note: I think we should consider `data-test="*"` since that is preferred by Cypress. `data-qa` is a qawolf thing.)

Reply:
Yes great point :grinning: I just added `data-qa` to the comment since we talked about it. It doesn't matter what we end up with just so long as it makes sense for us is what I'm thinking. For the moment `data-focus-path` should do the job!